### PR TITLE
test cleanup

### PR DESCRIPTION
**Pull request by tjlane**
Pulling some changes to testing that are just organization/cleanup out of #14 to make that PR more managable.

### CODE REVIEW

File: meteor/utils.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 class MapLabels:
     amplitude: str
     phase: str
+    error: str | None = None
```
Comment (alisiafadini) [inline meteor/utils.py:15]: wait, is error = uncertainty?

File: test/unit/conftest.py [hunk 2]
```diff
@@ -46,38 +47,41 @@ def generate_single_carbon_density(
...
         phase_label="PHIC",
     )
-    assert (difference_map_coefficients.max() > 0.0).any()
 
     return difference_map_coefficients
 
 
+@pytest.fixture
+def carbon_difference_density() -> np.ndarray:
+    difference_density = np.array(carbon1_density()) - np.array(carbon2_density())
```
Comment (alisiafadini) [inline test/unit/conftest.py:81]: what was the final verdict here? Ripples or no ripples?

File: meteor/testing.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+
+def assert_phases_allclose(array1: np.ndarray, array2: np.ndarray, atol=1e-3):
```
Comment (alisiafadini) [inline meteor/testing.py:4]: You probably have a good reason for this, but why have this in testing instead of utils?

**Review by alisiafadini — APPROVED** [review]

File: meteor/utils.py (hunk 1 above)
Comment (tjlane) [reply meteor/utils.py:15]: hmmm yes `uncertainty` (or `std`?) is a better name!

**Pull request merged**